lib/presentation/shared_widgets/participant_avatars.dart

```diff
@@ -34,7 +35,7 @@ class ParticipantAvatars extends StatelessWidget {
             ? CircleAvatar(
                 radius: 18,
                 backgroundColor: Colors.grey[300],
-                backgroundImage: NetworkImage(
+                backgroundImage: CachedNetworkImageProvider(
```

Comment:
Can we set the caching policy explicitly to invalidate after some time (like 24h) to show the updated profile pictures of other accounts eventually?

Reply:
Best option would be to leave them as plain `NetworkImage`

Reply:
Then I think we should do that